samples/cpp/benchmark_app/utils.cpp

```diff
@@ -767,6 +768,176 @@ void load_config(const std::string& filename, std::map<std::string, ov::AnyMap>&
 }
 #endif
 
+void dump_property(const std::string& filename, const std::map<std::string, ov::AnyMap>& property) {
+    nlohmann::json jsonConfig;
+    std::map<std::string, PropertyInfo> property_map;
+    std::map<std::string, std::string> enum_map;
+    generate_property_map(property_map, true);
+    generate_enum_map(enum_map);
+    for (const auto& item : property) {
+        std::string deviceName = item.first;
+        for (const auto& option : item.second) {
+            auto key = option.first;
+            std::stringstream strm;
+            option.second.print(strm);
+            auto value = strm.str();
+            auto iter = property_map.find(key);
+            if (iter == property_map.end()) {
+                slog::warn << "Not support property:" << key << slog::endl;
+                continue;
+            }
+            std::string property_key = iter->second.property_name;
+            PropertyType property_type = iter->second.property_type;
+            if (property_type == PROPERTY_TYPE_BOOL) {
+                if (value == "YES") {
+                    jsonConfig[deviceName][property_key] = true;
+                } else {
+                    jsonConfig[deviceName][property_key] = false;
+                }
+            } else if (property_type == PROPERTY_TYPE_ENUM) {
+                jsonConfig[deviceName][property_key] = value;
+                for (auto& element : enum_map) {
+                    if (element.second == value) {
+                        jsonConfig[deviceName][property_key] = element.first;
+                        break;
+                    }
+                }
+            } else if (property_type == PROPERTY_TYPE_STRING_FLOAT_MAP) {
+                std::map<std::string, float> out_map;
+                out_map = ov::util::from_string<std::map<std::string, float>>(value);
+                for (auto& out : out_map) {
+                    jsonConfig[deviceName][property_key][out.first] = out.second;
+                }
+            } else if (property_type == PROPERTY_TYPE_FLOAT) {
+                jsonConfig[deviceName][property_key] = ov::util::from_string<float>(value);
+            } else if (property_type == PROPERTY_TYPE_INTER) {
+                jsonConfig[deviceName][property_key] = ov::util::from_string<int32_t>(value);
+            } else if (property_type == PROPERTY_TYPE_UNSIGNED_INTER) {
+                jsonConfig[deviceName][property_key] = ov::util::from_string<uint32_t>(value);
+            } else if (property_type == PROPERTY_TYPE_INTER64) {
+                jsonConfig[deviceName][property_key] = ov::util::from_string<int64_t>(value);
+            } else if (property_type == PROPERTY_TYPE_STRING) {
+                jsonConfig[deviceName][property_key] = value;
+            } else {
+                slog::warn << "Invalid property type:" << property_type << " for property:" << property_key
+                           << slog::endl;
+            }
```

Comment:
Use of `switch` statement is more convenient.

Reply:
Ok, I will modify it.